app/controllers/classifier.coffee

```diff
@@ -90,6 +93,8 @@ class Classifier extends Controller
     @subjectViewer.setClassification @classification
     @animalSelector.setClassification @classification
 
+    AnalyticsLogger.logEvent User.current?.zooniverse_id,subject.zooniverseId,'view',''
```

Comment:
i see your point. so this will generate events for the image shown on the homepage not just ones they are shown to classify.